README.md

```diff
@@ -233,12 +233,12 @@ git subtree add --squash --prefix=vendor/github.com/orgname/xyzzy/ vendor-xyzzy
 The [`scripts/presubmit.sh`](scripts/presubmit.sh) script runs various tools
 and tests over the codebase.
 
-```bash
 # Install [golangci-lint](https://github.com/golangci/golangci-lint#local-installation).
```

Comment:
By moving the code block markers around this is now a top-level heading.

Reply:
Further down also has a top-level heading that shouldn't be there, and the wrong code/non-code style for the rest of the doc.

Best to preview Markdown files...

Reply:
That wasn't super obvious.  Sorry about that.